### PR TITLE
Fix fb rotation on the Forma w/ KSM

### PR DESCRIPTION
**Pull request by NiLuJe**
KSM is currently emulating a legacy pickel rotation (UR, 0), instead of what both pickel & Nickel do (CCW, 3) on the Forma.

Besides the bad interaction with FBInk described in #4752, I found out that the eInk driver was doing very weird things with A2 updates when UR @ 8bpp...

### CODE REVIEW

**Comment by NiLuJe** [comment]
Pinging @ptrm, as I have no way to check if behaves as intended on KSM ;).

**Comment by NiLuJe** [comment]
The most user-visible change as far as A2 quirks are concerned would be that FORCE_MONOCHROME will actually take now, so you won't get crappy fringes around icons when they're inverted.

File: platform/kobo/koreader.sh [hunk 1]
```diff
@@ -217,6 +217,17 @@ if [ -n "${ORIG_FB_BPP}" ]; then
     ./fbdepth -d "${ORIG_FB_BPP}" >>crash.log 2>&1
 fi
 
+# Restore original fb rotation on the Forma, for KSM
+if [ "${PRODUCT}" = "frost" ]; then
+    # Only needed for KSM, pickel -> Nickel will restore its own rota properly
+    if [ "${FROM_NICKEL}" != "true" ]; then
+        # The Forma kernel inverts odd rotation constants, counteract that, àla Plato because one-liner ;p.
```
Comment (Frenzie) [inline platform/kobo/koreader.sh]: I'm oddly bothered by that à la without a space. :-P

Comment (NiLuJe) [reply platform/kobo/koreader.sh]: That's my own bad habit, AFAICT ;).

**Comment by ptrm** [comment]
In general, the new `koreader.sh` works, and regardless of later orientation the touches correspond with the screen reaction. Did not have time to downgrade back to previous nightly to check if FBInk status overlay is in place, probably it will turn out tomorrow ;)

**Comment by NiLuJe** [comment]
@ptrm: Thanks! I assume KSM's rotated properly on exit?

**Comment by ptrm** [comment]
>I assume KSM's still rotated properly when KOReader exits?

That's right, and touch response is proper as well

**Comment by NiLuJe** [comment]
Off we go, then, thanks!

**Comment by ptrm** [comment]
FTR, FBInk was oriented in portrait during OTA update syncing and unpacking today

**Comment by NiLuJe** [comment]
\o/

**Pull request merged**